src/agoradatatools/great_expectations/gx/expectations/target_exp_validation_harmonized.json

```diff
@@ -0,0 +1,345 @@
+      },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_value_lengths_to_be_between",
+      "kwargs": {
+        "column": "outcome_measure_details",
+        "max_value": 2000,
+        "min_value": 2
+      },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_values_to_be_of_type",
+      "kwargs": {
+        "column": "balanced_for_sex",
+        "type_": "str"
+      },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_values_to_be_in_set",
+      "kwargs": {
+        "column": "balanced_for_sex",
+        "value_set": [
+          "yes",
+          "Yes",
+          "no",
+          "No"
+        ]
+      },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_values_to_be_of_type",
+      "kwargs": {
+        "column": "contributors",
+        "type_": "str"
+      },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_value_lengths_to_be_between",
+      "kwargs": {
+        "column": "contributors",
+        "max_value": 1000,
+        "min_value": 2
+      },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_values_to_be_of_type",
+      "kwargs": {
+        "column": "team",
+        "type_": "str"
+      },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_values_to_be_in_set",
+      "kwargs": {
+        "column": "team",
+        "value_set": [
+          "Duke",
+          "Columbia-Rush",
+          "MSSM - Zhang Lab",
+          "Emory",
+          "Mayo-UFL-ISB",
+          "Mayo"
+        ]
+      },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_values_to_be_of_type",
+      "kwargs": {
+        "column": "reference_doi",
+        "type_": "str"
+      },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_value_lengths_to_be_between",
+      "kwargs": {
+        "column": "reference_doi",
+        "max_value": 1000,
+        "min_value": 2
+      },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_values_to_match_regex",
+      "kwargs": {
+        "column": "reference_doi",
+        "regex": "https://doi\\.org/[A-Za-z0-9/]+"
+      },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_values_to_be_of_type",
+      "kwargs": {
+        "column": "date_report",
+        "type_": "str"
+      },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_values_to_match_strftime_format",
+      "kwargs": {
+        "column": "date_report",
+        "strftime_format": "%m/%d/%y"
+      },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_values_to_be_in_set",
+      "kwargs": {
+        "column": "date_report",
+        "value_set": [
+          "12/1/20",
+          "12/1/23"
+        ]
+      },
+      "meta": {}
+    }
+  ],
+  "ge_cloud_id": null,
+  "meta": {
+    "great_expectations_version": "0.18.1"
+  }
+}
```

Comment:
I can't recall if multifield-validation checks like these require custom expectations, but if they don't these would be nice to have:

  if (tolower(published) == "yes") !is.na(reference),
  if (tolower(published) == "yes") !is.na(reference_doi)

Reply:
This is similar to our previous [discussion](https://github.com/Sage-Bionetworks/agora-data-tools/pull/123#discussion_r1496258874). I think it would require a custom expectaion.

Reply:
tracking this via AG-1390 for the future